musicbot/bot.py

```diff
@@ -3129,6 +3132,41 @@ def _get_url() -> str:
                 )
             return url
 
+        if option in ["+all", "add_all"]:
+            if not player.playlist.entries:
+                raise exceptions.CommandError(
+                    self.str.get(
+                        "cmd-autoplaylist-add-all-empty-queue",
+                        "The queue is empty. Add some songs with `{0}play`!",
+                    ).format(self.server_data[guild.id].command_prefix),
+                    expire_in=30,
+                )
+
+            added_songs = set()
+            for e in player.playlist.entries:
+                url = e.url
+                self._do_song_blocklist_check(url)
```

Comment:
We can probably avoid doing blocklist checking for this case.  In theory, the songs that have made it into the queue have already been checked.  (Just not 100% sure about playlist entries, but they should be too.)